openapi.json

```diff
@@ -2913,7 +2913,7 @@
             }
           }
         ],
-        "description": "Pipeline configuration specified by the user when creating\na new pipeline instance.\nTODO: change this description\n\nThis is the shape of the overall pipeline configuration. It encapsulates a\n[`RuntimeConfig`], which is the publicly exposed way for users to configure\npipelines."
+        "description": "Overall pipeline configuration.\n\nIt is generated upon the deployment of a pipeline and contains\nthe shape of the overall pipeline configuration.\n\nIts input and output endpoints are generated based on the schema\nof the compiled program. The runtime configuration is directly\nprovided by the user. Storage configuration, if applicable,\nis set by the runner."
```

Comment:
Is this for the PipelineConfig? The first two sentences are redundant and don't tell the user much about the object. Just say: "Pipeline's runtime configuration, including the number of workers, resource allocations, and connectors used. It represents configuration entries provided by the user (...) and entries derived from the program and system deployment (connectors...)"

Reply:
It is part of the OpenAPI spec because it is a generated field `deployment_config`, which the user has in the extended pipeline configuration. I agree I wrote it a bit verbose, will adjust the text :+1: